### PR TITLE
Update Human.php

### PR DESCRIPTION
**Pull request by ZenaGamingsky**
skin fix

### CODE REVIEW

**Comment by DRedDogPE** [comment]
Have you tested this?

**Comment by Driesboy** [comment]
no

File: src/pocketmine/entity/Human.php [hunk 1]
```diff
@@ -184,43 +186,51 @@ public function saveNBT(){
...
-				throw new \InvalidStateException((new \ReflectionClass($this))->getShortName() . " must have a valid skin set");
-			}
+//			if(strlen($this->skin) < 64 * 32 * 4){
+//				$this->server->getLogger()->info((new \ReflectionClass($this))->getShortName() . " must have a valid skin set");
+//			}
 
 
-			if($this instanceof Player){
-				$this->server->updatePlayerListData($this->getUniqueId(), $this->getId(), $this->getName(), $this->isSlim, $this->skin, [$player], $this->skinflag);
+			if(!($this instanceof Player)){
```
Comment (Subtleoliver) [inline src/pocketmine/entity/Human.php]: Why?

**Comment by ZenaGamingsky** [comment]
yes but it only shows arms

**Comment by keithkfng** [comment]
Exactly. I believe we've already fixed skins.

**Pull request closed without merging**